recipes/volume_extend.rb

```diff
@@ -0,0 +1,63 @@
+#
```

Comment:
i ask because its not there :p
https://github.com/Seth-Karlo/chef-gluster/blob/master/recipes/server.rb#L24